CDCgov/prime-reportstream
# Add external name to receiver

This PR adds the external name property to the Receiver for display on the website.

In order to test: 
1. Pull down the branch
2. Run unit tests. Everything should be happy
3. Run `end2end`. Everything should be happy
4. Examine the database to look at the latest record for IGNORE.CSV. The JSON should include the externalName property

## Changes
- Adds the `externalName` property to the receiver
- Updates the organization file to include the property for testing locally
- Adds three simple tests for the receiver (yay code coverage)
- Fixes a mess of formatting errors

## Checklist

### Testing

### Process

## Fixes
*List GitHub issues this PR fixes*
- #1241 

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue
